### ISSUE TITLE
React HOOKS API: useRef doesn't work

### ISSUE THREAD

**Issue by da45** [reactstrap/reactstrap#1414]
- reactstrap version `#7.1.0`
in 16.8 React version, Hooks are arriving. thanks to hooks we can use state without class,

```
import React, {useRef } from 'react';
let loginEl = useRef();

<Input  ref={loginEl} type="text" placeholder="your name ..." />
```

with Reactstrap Input component we lose ref, and `loginEl.current.value` is undefined,

**Comment by TheSharpieOne** [comment]
Probably will be resolved with #1356

**Comment by da45** [comment]
> Probably will be resolved with #1356

yes it works with innerRef found in the link you provided, thank you,
